docs/developers/api.md

```diff
@@ -10,3 +10,7 @@ Example of these apps include:
 
 The documentation of the API can be found [here](https://apidoc.hermez.network/)
 
+Hermez Testnet API can be found [here](https://api.testnet.hermez.io/v1)
```

Comment:
- I would add a link that returns the state of the network
- I would set API Mainnet before Testnet API 

```suggestion
Hermez Testnet API can be found [here](https://api.testnet.hermez.io/v1/state)
```